src/components/AIToolsContainer/AIToolsContainer.tsx

```diff
@@ -22,6 +26,18 @@ const AIToolsContainer = ({}: AIToolsContainerProps) => {
     return `${baseUrl}${encodeURIComponent(message + prompt)}`;
   };
 
+  const handleCopyMarkdown = async () => {
+    const markdown = await fetch(copyAsMarkdownProvider.baseUrl).then(
+      (response) => response.text(),
+    );
+    navigator.clipboard.writeText(markdown);
```

Comment:
this is a promise that can fail in some cases (usually if you try to programatically copy text, but this is onClick, so it's fine). but should discard the result either way to prevent an unhandled promise rejection:

```suggestion
    void navigator.clipboard.writeText(markdown);
```